protocol/avdl/keybase1/teams.avdl

```diff
@@ -269,6 +269,10 @@ protocol teams {
     // Keyed by per-team-key generation
     map<PerTeamKeyGeneration, PerTeamKey> perTeamKeys;
 
+    // Creation time of latest PerTeamKey, based on CTime from the
+    // link key appeared in.
+    UnixTime perTeamKeyCTime;
```

Comment:
Note that this may be zero instead of correct as long as diskStorageVersion=9 because the cache has not been busted.

Reply:
I did change the version though, but maybe we don't have to and we can live with 0-value here? worst case is that there clients will do rotations for a bit when they could be skipped